### PR TITLE
Feat/create new roles

### PR DESCRIPTION
**Pull request by Storti-Scatambuli**
Created new user roles: Curator, Author and Specialist

### CODE REVIEW

File: cifonauta/settings.py [hunk 1]
```diff
@@ -96,6 +98,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'cebimar',
         'USER': 'nelas',
+        'PASSWORD': '123'
```
Comment (RianPolonini) [inline cifonauta/settings.py]: This setting is only for your development environment, so you need to undo this change and be careful not to commit it again.

**Review by RianPolonini — CHANGES_REQUESTED** [review]

File: cifonauta/settings.py (hunk 1 above)
Comment (Storti-Scatambuli) [reply cifonauta/settings.py]: resolved

**Review by RianPolonini — APPROVED** [review]
🚀
